src/routes/spotify.ts

```diff
@@ -53,7 +54,7 @@ router.get("/authentication-callback", async (req, res) => {
   req.session.authentication_state = undefined;
 
   // Pull out origin to redirect to and clear it
-  const originToRedirectTo = req.session.authentication_origin;
+  let originToRedirectTo = req.session.authentication_origin;
```

Comment:
This can stay `const`

Reply:
cool, I was playing around with it adding logs, etc and forgot to clean up!